<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransactionDetailsDialog</class>
 <widget class="QDialog" name="TransactionDetailsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>739</width>
    <height>296</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Transaction details</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>12</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="m_transactionDetailsTabBar">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="m_transactionTab">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
       <property name="leftMargin">
        <number>35</number>
       </property>
       <property name="topMargin">
        <number>30</number>
       </property>
       <property name="rightMargin">
        <number>25</number>
       </property>
       <property name="bottomMargin">
        <number>25</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,1">
         <property name="spacing">
          <number>10</number>
         </property>
         <item>
          <widget class="WalletGui::WalletLargeBlackTextLabel" name="m_textLabel">
           <property name="text">
            <string>Transaction</string>
           </property>
           <property name="indent">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="WalletGui::WalletLargeBlackTextLabel" name="m_hashLabel">
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="toolTip">
            <string>Click to copy</string>
           </property>
           <property name="text">
            <string>HASH</string>
           </property>
           <property name="indent">
            <number>0</number>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignLeft">
          <widget class="WalletGui::CopyMagicLabel" name="m_copyLabel">
           <property name="text">
            <string>Copied!</string>
           </property>
           <property name="indent">
            <number>0</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout" columnstretch="0,1">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="WalletGui::WalletNormalGrayTextLabel" name="m_statusTextLabel">
           <property name="text">
            <string>STATUS</string>
           </property>
           <property name="indent">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="WalletGui::WalletNormalBlackTextLabel" name="m_statusLabel">
           <property name="text">
            <string>TextLabel</string>
           </property>
           <property name="indent">
            <number>0</number>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="WalletGui::WalletNormalGrayTextLabel" name="m_timeTextLabel">
           <property name="text">
            <string>TIME</string>
           </property>
           <property name="indent">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="WalletGui::WalletNormalBlackTextLabel" name="m_timeLabel">
           <property name="text">
            <string>TextLabel</string>
           </property>
           <property name="indent">
            <number>0</number>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="WalletGui::WalletNormalGrayTextLabel" name="m_heightTextLabel">
           <property name="text">
            <string>BLOCK HEIGHT</string>
           </property>
           <property name="indent">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="WalletGui::WalletNormalBlackTextLabel" name="m_heightLabel">
           <property name="text">
            <string>TextLabel</string>
           </property>
           <property name="indent">
            <number>0</number>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="WalletGui::WalletNormalGrayTextLabel" name="m_toTextLabel">
           <property name="text">
            <string>TRANSFERS</string>
           </property>
           <property name="indent">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="WalletGui::WalletNormalBlackTextLabel" name="m_toLabel">
           <property name="text">
            <string>TextLabel</string>
           </property>
           <property name="indent">
            <number>0</number>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="WalletGui::WalletNormalGrayTextLabel" name="m_amountTextLabel">
           <property name="text">
            <string>AMOUNT</string>
           </property>
           <property name="indent">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="WalletGui::WalletNormalBlackTextLabel" name="m_amountLabel">
           <property name="text">
            <string>TextLabel</string>
           </property>
           <property name="indent">
            <number>0</number>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="WalletGui::WalletNormalGrayTextLabel" name="m_feeTextLabel">
           <property name="text">
            <string>FEE</string>
           </property>
           <property name="indent">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="WalletGui::WalletNormalBlackTextLabel" name="m_feeLabel">
           <property name="text">
            <string>TextLabel</string>
           </property>
           <property name="indent">
            <number>0</number>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="WalletGui::WalletNormalGrayTextLabel" name="m_paymentIdTextLabel">
           <property name="text">
            <string>PAYMENT ID</string>
           </property>
           <property name="indent">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="WalletGui::WalletNormalBlackTextLabel" name="m_paymentIdLabel">
           <property name="text">
            <string>TextLabel</string>
           </property>
           <property name="indent">
            <number>0</number>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="m_transfersTab">
      <attribute name="title">
       <string>Transfers</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="leftMargin">
        <number>25</number>
       </property>
       <property name="topMargin">
        <number>30</number>
       </property>
       <property name="rightMargin">
        <number>25</number>
       </property>
       <property name="bottomMargin">
        <number>25</number>
       </property>
       <item>
        <widget class="WalletGui::WalletTreeView" name="m_transfersView">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::NoSelection</enum>
         </property>
         <property name="textElideMode">
          <enum>Qt::ElideMiddle</enum>
         </property>
         <property name="verticalScrollMode">
          <enum>QAbstractItemView::ScrollPerPixel</enum>
         </property>
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <property name="uniformRowHeights">
          <bool>true</bool>
         </property>
         <property name="itemsExpandable">
          <bool>false</bool>
         </property>
         <attribute name="headerStretchLastSection">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WalletGui::WalletTreeView</class>
   <extends>QTreeView</extends>
   <header>Gui/Common/WalletTreeView.h</header>
  </customwidget>
  <customwidget>
   <class>WalletGui::CopyMagicLabel</class>
   <extends>QLabel</extends>
   <header>Gui/Common/CopyMagicLabel.h</header>
  </customwidget>
  <customwidget>
   <class>WalletGui::WalletNormalGrayTextLabel</class>
   <extends>QLabel</extends>
   <header>Gui/Common/WalletTextLabel.h</header>
  </customwidget>
  <customwidget>
   <class>WalletGui::WalletNormalBlackTextLabel</class>
   <extends>QLabel</extends>
   <header>Gui/Common/WalletTextLabel.h</header>
  </customwidget>
  <customwidget>
   <class>WalletGui::WalletLargeBlackTextLabel</class>
   <extends>QLabel</extends>
   <header>Gui/Common/WalletTextLabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
